apps/condo/domains/common/components/EmptyListContent.tsx

```diff
@@ -35,12 +35,15 @@ type IEmptyListWithoutImportProps = IBaseEmptyListProps & {
     createLabel?: string
 }
 
+type TExternalImportWrapper = Pick<IImportWrapperProps, 'onFinish' | 'uploadButtonLabel' | 'importCardButton'>
```

Comment:
For better compatibility and understanding
```suggestion
type ExternalImportWrapperType = Pick<IImportWrapperProps, 'onFinish' | 'uploadButtonLabel' | 'importCardButton'>
```

Reply:
The **T** in the beginning means the same :)